openstack/02b-MAChigh/neighbors.c

```diff
@@ -543,7 +543,8 @@ void neighbors_updateMyDAGrankAndNeighborPreference() {
             rankIncrease = (uint16_t)(rankIncreaseIntermediary >> 10);
          }
          
-         tentativeDAGrank = neighbors_vars.neighbors[i].DAGrank+rankIncrease;
+         tentativeDAGrank = neighbors_vars.neighbors[i].DAGrank;
+         tentativeDAGrank += rankIncrease;

```

Comment:
I will add a short comment to the patch to avoid that someone "unfixes" this later. (I think travis can handle another build...)